.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```

Comment:
Great job @Bludidi 🤝

- [ ] Great job adding the `linters.yml file`. You may also add the `.hintrc` file and `stylintrc.json` file in the root directory. The files can be found in the correct [linter files](https://github.com/microverseinc/linters-config/tree/master/html-css) provided.

Reply:
I have added .hintrc file and stylintrc.jason